tests/Support/SupportCollectionTest.php

```diff
@@ -2421,6 +2422,21 @@ public function testRangeMethod($collection)
             [-2, -3, -4],
             $collection::range(-2, -4)->all()
         );
+
+        $this->assertSame(
+            [0, 2, 4, 6, 8, 10],
+            $collection::range(0, 10, 2)->all()
+        );
+
+        $this->assertSame(
+            [-5, -3, -1, 1, 3, 5],
+            $collection::range(-5, 5, 2)->all()
+        );
+
+        $this->assertSame(
+            [0, 5],
+            $collection::range(0, 5, 5)->all()
+        );
```

Comment:
You test when `$from > $to`, but neither when `$from == $to` nor when `$from < $to`